clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -301,20 +302,33 @@ object GarbageCollector {
   }
 
   def main(args: Array[String]) {
-    if (args.length != 2) {
+    if (args.length < 1) {
```

Comment:
I find this confusing... as a user running GC on azure, I will be confused in the following case:
arguments: empty, and the application tells me that I need to use an optional "region", but it is not really optional for Azure 

Can we add a condition special to Azure in an else if at line 326 and remove this one?